src/XrdCeph/XrdCephBulkAioRead.hh

```diff
@@ -0,0 +1,80 @@
+#include <string>
+#include <map>
+#include <list>
+#include <tuple>
+#include <rados/librados.hpp>
+
+
+class bulkAioRead {
+  //typedef std::tuple<ceph::bufferlist*, char*, int*> ReadOpData;
+  typedef void (*logfunc_pointer) (char *, ...);
+
+  /**
+   * Class is used to execute read operations against rados striper files *without* usage of rados striper.
+   * Reads are based on ceph read operations.
+   *
+   * The interface is similar to the one that ceph's read operation objects has:
+   * 1. Instantiate the object.
+   * 2. Declare read operations using 'read' method, providing the output buffers, offset and length.
+   * 3. Submitn operation and wait for results using 'submit_and_wait_for_complete' method.
+   * 4. Copy results to buffers with 'get_results' method. 
+   *
+   * WARNING: there is no copy/move constructor in the class, so do not use temporary objects for initialization
+   * (i.e. something like `bulkAioRead rop = bulkAioRead(...);` will not work, use `bulkAioRead rop(...);` instead).
+   */ 
+  public:
+  bulkAioRead(librados::IoCtx *ct, logfunc_pointer ptr, std::string filename, size_t object_size);
+  bulkAioRead(bulkAioRead&& tmp);
```

Comment:
Looks like this constructor has no implementation.  Perhaps add `= delete`?

Reply:
Sorry, my bad. I was experimenting with this move constructor and forgot to remove it afterwards.
Fixed.